plugin/src/main/kotlin/com/ritense/plugin/web/rest/PluginConfigurationResource.kt

```diff
@@ -98,6 +100,11 @@ class PluginConfigurationResource(
     @GetMapping("/v1/plugin/configuration/export")
     fun exportPluginConfiguration(): ResponseEntity<List<PluginConfigurationExportDto>> {
         val pluginConfigurations = pluginService.getPluginConfigurations(PluginConfigurationSearchParameters())
+            .sortedWith(comparingInt<PluginConfiguration> { pluginConfiguration ->
+                pluginConfiguration.properties?.fieldNames()?.asSequence()?.count { it.contains("PluginConfiguration") } ?: 0
```

Comment:
What if we have triple-nested configurations? I don't think you can get away with just sorting then. You'd have to follow the references somehow.

Reply:
Its true that this approach wont work for future plugins. But it works for all current plugins. imo its good enough for a hidden endpoint. 

But if you're in need of a challenge, you can give it a shot :)

Reply:
Challenge accepted!